src/tcl/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+file(GLOB_RECURSE Espresso_SRC
+          "*.cpp"
+          )
+
+include_directories(${CMAKE_SOURCE_DIR}/src/core)
+include_directories(${CMAKE_BINARY_DIR}/src/core)
+include_directories(${CMAKE_SOURCE_DIR}/src/tcl)
```

Comment:
Don't need `${CMAKE_SOURCE_DIR}` here, `include_directories` takes paths relative to the current source directory. So you could use `../../core` and `../../tcl`.